### ISSUE TITLE
Need spinner while container downloads

### ISSUE THREAD

**Issue by kdemarest** [appsoma/welder#83]

**Comment by zsimpson** [comment]
Unfortunately I have no way of know this. Mesos gives me no feedback about what it is doing.  It might be possible to create a custom "Executor" which might know more, not sure. Also not sure if the customer executor will be compatible with future shift to singularity or chronos.

I can add a spinner on "STAGING" but then I guess all the states need spinners.

**Comment by kdemarest** [comment]
Well, all that really needs to happen is that the user needs to know his request wasn't ignored.

So maybe the right way is to just add the item requested to the list, but with a status of "booting" and a little button beside it of fa-refresh ?

**Comment by zsimpson** [comment]
Just realized that maybe you mean on the start container page not the run page. Which do you mean?

**Comment by kdemarest** [comment]
Yes, when starting a container.

Immediate feedback would look like this:

![image](https://cloud.githubusercontent.com/assets/2153005/8120679/9da29ba2-1067-11e5-8992-4c4eb2f34828.png)

**Comment by zsimpson** [comment]
Yeah, again, there's no database for this. So I could throw up something client side but of course a page reload would obliterate it.

**Comment by zsimpson** [comment]
Actualyl maybe I can get the stdout like i do on the burn.
